support-frontend/assets/pages/contributions-landing/components/ContributionForm.jsx

```diff
@@ -295,6 +330,19 @@ function withProps(props: PropTypes) {
         <ContributionFormFields />
         <PaymentMethodSelector />
 
+        {props.paymentMethod === Sepa && (
+          <>
```

Comment:
We can drop the `<> </>`

Reply:
there are actually 2 components inside this

Reply:
ahhh ofc! my bad! 🙈